config/app.php

```diff
@@ -50,6 +50,7 @@
         \Engelsystem\Middleware\SetLocale::class,
         \Engelsystem\Middleware\ETagHandler::class,
         \Engelsystem\Middleware\AddHeaders::class,
+        \Engelsystem\Middleware\TrimMiddleware::class,
```

Comment:
Instead of `TrimMiddleware` (which is a duplication as Middleware already exists in the namespace) i would suggest naming it something like `TrimInput` (or like in laravel `TrimStrings`)